deepeval/benchmarks/hellaswag/template.py

```diff
@@ -0,0 +1,28 @@
+from deepeval.benchmarks.hellaswag.task import HellaSwagTask
+
+
+class HellaSwagTemplate:
+
+    staticmethod
```

Comment:
Missing @ 

Also is it possible to leave a sentence or two on these benchmark template files to say where they were taken from/rationale behind how they were constructued?